localization/en.ftl

```diff
@@ -282,10 +282,10 @@ multiline-opt-multilinifyDelayAfterEdit-name = Edit Delay (ms)
 multiline-opt-multilinifyDelayAfterEdit-desc = Multiline expressions should be updated after no edits are made for this number of milliseconds.
 multiline-opt-spacesToNewlines-name = Spaces to Newlines
 multiline-opt-spacesToNewlines-desc = Convert groups of 3 spaces into newlines. These can be automatically created with Shift+Enter.
-multiline-opt-determineLineBreaksAutomatically-name = Auto Insert Linebreaks 
+multiline-opt-determineLineBreaksAutomatically-name = Auto Insert Linebreaks
```

Comment:
Do you have something automatically removing trailing spaces? That could be a separate PR to keep this PR smaller.

Reply:
Yeah, didn't mean to touch other translations sorry. Will undo.